opnode/rollup/derive/invert_test.go

```diff
@@ -37,17 +40,22 @@ func (l *l1MockInfo) MixDigest() common.Hash {
 	return l.mixDigest
 }
 
+func (l *l1MockInfo) ReceiptHash() common.Hash {
+	return l.receiptRoot
+}
+
 func randomHash(rng *rand.Rand) (out common.Hash) {
 	rng.Read(out[:])
 	return
 }
 
 func randomL1Info(rng *rand.Rand) *l1MockInfo {
 	return &l1MockInfo{
-		num:     rng.Uint64(),
-		time:    rng.Uint64(),
-		hash:    randomHash(rng),
-		baseFee: big.NewInt(rng.Int63n(1000_0000 * 1e9)), // a million GWEI
+		num:         rng.Uint64(),
+		time:        rng.Uint64(),
+		hash:        randomHash(rng),
+		baseFee:     big.NewInt(rng.Int63n(1000_0000 * 1e9)), // a million GWEI
```

Comment:
slight nit, shouldn't `1000_0000` be `1_000_000` for ` a million gwei`, also there is `params.Gwei` instead of `1e9`

https://github.com/ethereum/go-ethereum/blob/d8ff53dfb8a516f47db37dbc7fd7ad18a1e8a125/params/denomination.go#L26

Reply:
Good catch! Although not intended part of the PR. Will include it though, small fix